module-audio/Audio/AudioCommon.hpp

```diff
@@ -77,10 +77,17 @@ namespace audio
         Other
     };
 
-    enum class FadeIn
+    enum class Fade
     {
         Disable,
-        Enable
+        In,
+        InOut
+    };
+
+    struct FadeParams
+    {
+        Fade mode;
+        std::optional<std::chrono::seconds> playbackDuration = std::nullopt;
```

Comment:
`std::optional` defaults to `std::nullopt`, there's no need to initialize it explicitly

Reply:
done